3711/2/bot.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'telegram/bot'
+require './commands/start'
+require './commands/checkin'
+require './commands/checkout'
+
+class WebhooksController < Telegram::Bot::UpdatesController
+  include StartCommand
+  include CheckInCommand
+  include CheckOutCommand
+  include Telegram::Bot::UpdatesController::MessageContext
+
+  def initialize(*)
```

Comment:
`*` is redundant

Reply:
it does'n work in that case
added `args` after `*`